oracle/internal/dkg/dkg_executor.go

```diff
@@ -122,25 +124,25 @@ func (e *Executor) OnStartNewDKG(dkg *coordinator.DKG) bool {
 	{
 		sessionSigner, err := validator.NewSessionSigner(e.keystore, dkg.Until.Unix())
 		if err != nil {
-			e.logDKGProcess(dkg, fmt.Sprintf("Failed to create SessionSigner: %v", err))
+			e.logDKGProcess(dkg, e.currentValidatorIdx, fmt.Sprintf("Failed to create SessionSigner: %v", err))
```

Comment:
Since you store currentValidatorIdx inside Executor struct, you don't need to pass it separately. Use `e.currentValidatorIdx` inside log function.